components/MenuItem/MenuItem.js

```diff
@@ -152,8 +146,9 @@ const StyledSubNavigationListItem = styled.li`
 
 const StyledNavigationLink = styled(Link)`
   text-decoration: none;
-  color: ${({ disabled }) => (disabled ? "#838383" : "#fff")};
-  cursor: ${({ disabled }) => (disabled ? "not-allowed" : "pointer")};
+  color: ${({ $disabled }) => ($disabled ? "#838383" : "#fff")};
+  cursor: ${({ $disabled }) => ($disabled ? "not-allowed" : "pointer")};
+  pointer-events: ${({ $disabled }) => ($disabled ? "none" : "auto")};
```

Comment:
you could simplify this:
```suggestion
color: #fff;
cursor: pointer;
  
${({ $disabled }) => ($disabled && css`
  color: #838383;
  cursor:  not-allowed;
  pointer-events: none;
`)}
```

Reply:
Ah, thanks, the same applies here as above. I noticed the code repetition and thought about whether it could be made simpler, but my idea was to write an extra function for it, and that didn't really seemed like a good simplification to me. Your code is better. 😉